### PR TITLE
Fix minor typos

### PR DESCRIPTION
**Pull request by egnha**
#139 reloaded

### CODE REVIEW

File: vignettes/tidy-evaluation.Rmd [hunk 1]
```diff
@@ -144,19 +147,18 @@ addition to dataframe columns. In other words, the grammar implements
...
 combine expressions that do not necessarily come from the same lexical
 context. Secondly because arguments can be forwarded through the
-special `...` argument.  While base R does not provide any way of
+special `...` argument. While base R does not provide any way of
 capturing a forwarded argument along with its original environment,
 rlang features [quos()] for this purpose. This function looks up each
-forwarded arguments and returns a list
-of [quosures](http://rlang.tidyverse.org/reference/quosure.html) that
-bundle the expressions with their own dynamic environments.
+forwarded arguments and returns a list of [quosures] that bundle the
```
Comment (lionel-) [inline vignettes/tidy-evaluation.Rmd]: should be [quosures][quosure]

File: vignettes/tidy-evaluation.Rmd [hunk 2]
```diff
@@ -85,9 +87,10 @@ evaluation as much as possible.
 
 A corollary of emphasising evaluation is that your DSL functions
 should understand _values_ in addition to expressions. This is
-especially important with [quasiquotation]: users can bypass symbolic
-evaluation completely by unquoting values. For instance, the following
-expressions are completely equivalent:
+especially important with
+[quasiquotation](http://rlang.tidyverse.org/reference/quasiquotation.html):
```
Comment (lionel-) [inline vignettes/tidy-evaluation.Rmd]: `[quasiquotation]`

**Comment by egnha** [comment]
The spurious spaces were the doing of my editor's autofill. I have tried to eradicate them to make the changes clearer.

**Comment by lionel-** [comment]
no worries, thanks!

**Pull request merged**